lib/dockerregistry/inventory.go

```diff
@@ -2329,7 +2341,9 @@ func (sc *SyncContext) GarbageCollect(
 	if customProcessRequest != nil {
 		processRequest = *customProcessRequest
 	}
-	sc.ExecRequests(populateRequests, processRequest)
+	errorTracker := false
+
+	sc.ExecRequests(populateRequests, processRequest, &errorTracker)
```

Comment:
Let's at least log a "there was an error" or some such message if errorTracker is true in `GarbageCollect`. Likewise for `ClearRepository`.